policies/rudderc/tests/cases/general/ntp/technique.ps1

```diff
@@ -24,59 +24,30 @@
 
 
     $reportId=$reportIdBase + "d86ce2e5-d5b6-45cc-87e8-c11cca71d907"
-    $componentKey = "htop"
+    $componentKey = 'htop'
     $reportParams = @{
         ClassPrefix = ([Rudder.Condition]::canonify(("package_present_" + $componentKey)))
         ComponentKey = $componentKey
-        ComponentName = "Ensure correct ntp configuration"
+        ComponentName = 'Ensure correct ntp configuration'
         PolicyMode = $policyMode
         ReportId = $reportId
         DisableReporting = $false
         TechniqueName = $techniqueName
     }
-    
-    $class = "false"
-    if ($localContext.Evaluate($class)) {
-        $methodParams = @{
-            Architecture = ""
-            Name = "htop"
-            Provider = ""
-            Version = @'
-2.3.4
-'@
-            
-        }
-        $call = Package-Present @methodParams -PolicyMode $policyMode
-        $methodContext = Compute-Method-Call @reportParams -MethodCall $call
-        $localContext.merge($methodContext)
-    } else {
-        Rudder-Report-NA @reportParams
-    }
+    Rudder-Report-NA @reportParams
 
     $reportId=$reportIdBase + "cf06e919-02b7-41a7-a03f-4239592f3c12"
-    $componentKey = "/bin/true `"# ${node.inventory[os][fullName]}`""
+    $componentKey = '/bin/true "# ' + ([Nustache.Core.Render]::StringToString('{{' + 'node.inventory.os.fullName' + '}}', $data, $mustacheOptions)) + '"'
     $reportParams = @{
         ClassPrefix = ([Rudder.Condition]::canonify(("package_install_" + $componentKey)))
         ComponentKey = $componentKey
-        ComponentName = "NTP service"
+        ComponentName = 'NTP service'
         PolicyMode = $policyMode
         ReportId = $reportId
         DisableReporting = $false
         TechniqueName = $techniqueName
     }
-    
-    $class = "linux.fedora"
```

Comment:
this is quite unexpected that this class disapers if the source was not modified